src/init.cpp

```diff
@@ -235,18 +234,18 @@ void Shutdown(NodeContext& node)
         DumpMempool(*node.mempool);
     }
 
-    if (fFeeEstimatesInitialized)
+    if (node.feeEstimator)
     {
```

Comment:
Join lines to fix up with current code style:

`if (node.feeEstimator) {`

(see https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c)